classes/Tables/ilTagListTableGUI.php

```diff
@@ -0,0 +1,214 @@
+<?php
+
+namespace LC\ILP\GoogleAnalytics\Tables;
+
+use \LC\ILP\GoogleAnalytics\DataObjects\Tag;
+use \LC\ILP\GoogleAnalytics\DataObjects\TagCollection;
+
+/**
+ * Class ilTagListTableGUI
+ * @author Ralph Dittrich <dittrich@qualitus.de>
+ */
+class ilTagListTableGUI extends \ilTable2GUI
+{
+	/** @var \ilGoogleAnalyticsPlugin  */
+	protected $pl;
+
+	/** @var \ilLanguage  */
+	protected $lng;
+
+	/**
+	 * ilTagListTableGUI constructor.
+	 * @param \ilGoogleAnalyticsConfigGUI $parent_obj
+	 * @param string $parent_cmd
+	 */
+	public function __construct(\ilGoogleAnalyticsConfigGUI $parent_obj, string $parent_cmd = 'listTags')
+	{
+		global $DIC;
+
+		$this->pl = $parent_obj->getPlugin();
+		$this->lng = $DIC->language();
+
+		parent::__construct($parent_obj, $parent_cmd);
+
+		$this->setTitle($this->pl->txt('list_of_tags'));
+//		$this->setLimit(9999);
```

Comment:
Auskommentierten Code würde ich immer entfernen, zur Not haben wir die GIT History.

Reply:
Recht hast du. Ich entferne die Codestellen